src/scenes/HTTP/HttpDashboard.tsx

```diff
@@ -1,148 +1,60 @@
-                    <Stack justifyContent="space-between">
-                      <Stack direction="row" gap={2}>
-                        <VariableControl name="probe" />
-                        <DataLayerControl name="Alerts firing" />
-                        <DataLayerControl name="Alerts pending" />
-                      </Stack>
-                      <Stack direction="row" gap={2}>
-                        <EditCheckButton id={check.id} />
-                        <TimeRangePicker />
-                        <RefreshPicker />
-                      </Stack>
-                    </Stack>
-                    <div className={styles.vizLayout}>
-                      <div className={styles.errorRateMap}>
-                        <ErrorRateMap minStep={minStep} />
-                      </div>
+    <DashboardContainer check={check} checkType={CheckType.HTTP}>
+      <div className={styles.vizLayout}>
+        <div className={styles.errorRateMap}>
+          <ErrorRateMap minStep={minStep} />
+        </div>
 
-                      <div className={styles.nestedGrid}>
-                        <div className={styles.statsRow}>
-                          <UptimeStat check={check} />
-                          <ReachabilityStat minStep={minStep} />
-                          <AvgLatency />
-                          <SSLExpiry />
-                          <Frequency />
-                        </div>
+        <div className={styles.nestedGrid}>
+          <div className={styles.statsRow}>
+            <UptimeStat check={check} />
+            <ReachabilityStat check={check} />
+            <AvgLatency />
+            <SSLExpiry />
+            <Frequency />
+          </div>
 
-                        <ErrorRate minStep={minStep} />
-                      </div>
+          <ErrorRate minStep={minStep} />
+        </div>
 
-                      <div className={styles.latencyRow}>
-                        <div className={styles.latencyPanel}>
-                          <ResponseLatency />
-                        </div>
-                        <div className={styles.latencyPanel}>
-                          <ResponseLatencyByProbe />
-                        </div>
-                      </div>
+        <div className={styles.latencyRow}>
+          <div className={styles.latencyPanel}>
+            <ResponseLatency />
+          </div>
+          <div className={styles.latencyPanel}>
+            <ResponseLatencyByProbe />
+          </div>
+        </div>
 
-                      <div className={styles.errorLogs}>
-                        <ErrorLogs />
-                      </div>
-                    </div>
-                  </AnnotationLayer>
-                </AnnotationLayer>
-              </Stack>
-            </PluginPage>
-          </CustomVariable>
-        </CustomVariable>
-      </QueryVariable>
-    </SceneContextProvider>
+        <div className={styles.errorLogs}>
+          <ErrorLogs startingUnsuccessfulOnly={true} />
```

Comment:
```suggestion
          <ErrorLogs />
```

The old version shows all logs and not just unsuccessful ones by default. Not sure if this was on purpose.

Reply:
Yeah, this was on purpose to standardise them. I found it annoying that the behaviour was different between the API Endpoint checks and scripted / browser.

Reply:
For Browser and Scripted checks I see it shows all logs, not just failed ones though. IMO seeing all logs can still be useful so I'd leave it unchecked. 
Not that big of a deal anyway as this is soon going to change with your custom logs work.